kitsune/community/signals.py

```diff
@@ -8,12 +8,28 @@
 from kitsune.wiki.models import Document, Revision
 
 
+def is_marked_for_deletion(user, origin):
+    """
+    Within the context of a pre or post-delete event, returns whether or
+    not the deletion of the given user is at least part of the origin of
+    the event.
+    """
+    if not origin:
+        return False
+
+    if isinstance(origin, QuerySet):
+        return user in origin
+
+    # The origin is a Django model instance.
+    return user == origin
+
+
 @receiver(
     pre_delete,
     sender=Answer,
     dispatch_uid="community.record_deleted_answer_contributions",
 )
-def record_deleted_answer_contributions(sender, instance, **kwargs):
```

Comment:
I just realized that the current implementation will also count spam as contribution as long as the spammer has an account.